recipes/bowtie2/2.2.4/bowtie2_wrapper.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+usage() {
+      echo ""
+      echo "Usage : sh $0 -x <reference_index> -i Input_folder {-1 <left_reads> -2 <right_reads> | -U <single_reads>} -S <output_sam>"
+      echo ""
+
+cat <<'EOF'
+  -x reference index	
+
+  -i </path/to/input folder>
+
+  -1 </path/to/reads_1>
+
+  -2 </path/to/reads_2>
+
+  -U </path/to/single_reads>
+
+  -S </path/to/sam output>
+
+EOF
+    exit 0
+}
+
+while getopts ":hx:i:1:2:U:S:" opt; do
+  case $opt in
+    x)
+    ref_index=$OPTARG # Reference genome index
+     ;;
+    i)
+    input_folder=$OPTARG # Input folder
+     ;;
+    1)
+    left_reads=$OPTARG # Left reads
+     ;;
+    2)
+    right_reads=$OPTARG # Right reads
+     ;;
+    U)
+    single_reads=$OPTARG # single end reads
+     ;;
+    S)
+    sam_out=$OPTARG # Samoutput file
+     ;;
+    h)
+    usage
+     exit 1
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+    :)
+      echo "Option -$OPTARG requires an argument." >&2
+      exit 1
+      ;;
+  esac
+done
+
+export BOWTIE2_INDEXES=$input_folder
+
+if [ ! -z $left_reads ] && [ ! -z $right_reads ]; 
+then
+    /usr/local/bin/bowtie2 -x $ref_index -1 $left_reads -2 $right_reads -S $sam_out
```

Comment:
Without this shell script, the bowtie2 does not work in our Discovery Environment